src/sb_utils.R

```diff
@@ -29,4 +47,58 @@ sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
   
   sb_replace_files(sb_id = sb_id, xml_file)
   
-}
\ No newline at end of file
+}
+
+# Helper function to create a task_table for the files that need to be pushed to SB
+do_item_replace_tasks <- function(sb_id, files) {
+  
+  # Define task table columns
+  sb_push <- scipiper::create_task_step(
+    step_name = 'push_file_to_sb',
+    target_name = function(task_name, step_name, ...){
+      sprintf("%s_pushed_to_sb", task_name)
+    },
+    command = function(task_name, ...){
+      sprintf("upload_and_record(I('%s'), '%s')", sb_id, task_name)
+    } 
+  )
+  
+  # Create the task plan
+  task_plan <- create_task_plan(
+    task_names = files, # Define task table rows
+    task_steps = list(sb_push),
+    final_steps = c('push_file_to_sb'),
+    add_complete = FALSE)
+  
+  # Create the task remakefile
+  task_yml <- "file_upload_tasks.yml"
+  final_target <- "upload_timestamps"
+  create_task_makefile(
+    task_plan = task_plan,
+    makefile = task_yml,
+    packages = c('sbtools', 'scipiper', 'purrr'),
+    final_targets = final_target,
```

Comment:
now that you are using something outside of `sbtools` for the upload function, you need `sources`

Reply:
Should sources be passed in somehow from top-level? For the purposes of reusing this function code elsewhere, it might be sneaky to just add `sources = c("src/sb_utils.R")` inside of this function.

Reply:
agreed, but it should be a named argument, so as to not confuse with `...` files arg. I agree that it should be specifed outside in the `remake.yml`

Reply:
Hmmm seems tricky to do this and have it be elegant. What do you think of the latest commit?

Reply:
Alternatively, if you just did `include = "remake.yml"` inside of this function, wouldn't the src R script be included since it is specified in remake.yml in order to use `sb_replace_files` ?

Reply:
I suppose that assumes that all ymls that use `sb_replace_files` are linked via the top-level `remake.yml`

Reply:
No, they don't need to be linked - they point to existing files, which can be specified in a stand alone task yaml makefile. 

If possible, I'd try not to include the parent in this task unless it is unavoidable.